js/bitpanda.js

```diff
@@ -424,43 +426,19 @@ module.exports = class bitpanda extends Exchange {
         //         }
         //     ]
         //
-        const feeGroupsById = this.indexBy (response, 'fee_group_id');
-        const feeGroupId = this.safeValue (this.options, 'fee_group_id', 'default');
-        const feeGroup = this.safeValue (feeGroupsById, feeGroupId, {});
-        const feeTiers = this.safeValue (feeGroup, 'fee_tiers');
+        const feeTiers = this.safeValue (response, 'fee_tiers');
         const result = {};
         for (let i = 0; i < this.symbols.length; i++) {
             const symbol = this.symbols[i];
-            const fee = {
-                'info': feeGroup,
+            result[symbol] = {
+                'info': response,
                 'symbol': symbol,
                 'maker': undefined,
                 'taker': undefined,
                 'percentage': true,
                 'tierBased': true,
+                'tiers': this.parseFeeTiers (feeTiers),
             };
-            const takerFees = [];
-            const makerFees = [];
-            for (let i = 0; i < feeTiers.length; i++) {
-                const tier = feeTiers[i];
-                const volume = this.safeNumber (tier, 'volume');
-                let taker = this.safeNumber (tier, 'taker_fee');
-                let maker = this.safeNumber (tier, 'maker_fee');
-                taker /= 100;
-                maker /= 100;
-                takerFees.push ([ volume, taker ]);
-                makerFees.push ([ volume, maker ]);
-                if (i === 0) {
-                    fee['taker'] = taker;
```

Comment:
We are losing an important thing here, the default maker/taker fees are tier[0] fees. If possible, please restore those `maker` and `taker` values.